.lfsconfig

```diff
@@ -0,0 +1,2 @@
+[lfs]
+	url = https://cd2545dc-eb10-4a2a-a0c7-f9e2440e6bd2.netlify.app/.netlify/large-media
```

Comment:
Is this indentation intended?

Reply:
This is an autogenerated file, so probably yes